packages/instant/src/index.umd.ts

```diff
@@ -108,15 +108,15 @@ export interface ZeroExInstantConfig extends ZeroExInstantOverlayProps {
     shouldDisablePushToHistory?: boolean;
 }
 
-export const render = (config: ZeroExInstantConfig, selector: string = DEFAULT_ZERO_EX_CONTAINER_SELECTOR) => {
```

Comment:
I couldn't find where this was being called, so didn't rename it to `renderAsync`. Worried this breaks something else. Where is this used? @fragosti @steveklebanoff